corsheaders/tests.py

```diff
@@ -183,155 +182,184 @@ def assertAccessControlAllowOriginEquals(self, response, header):
-        processed = self.middleware.process_response(request, response)
-        self.assertEqual(processed.get(ACCESS_CONTROL_ALLOW_CREDENTIALS, None), 'true')
-
-    def test_process_response_adds_origin_when_domain_found_in_origin_regex_whitelist(self, settings):
-        settings.CORS_MODEL = None
-        settings.CORS_ORIGIN_REGEX_WHITELIST = ('^http?://(\w+\.)?google\.com$', )
-        settings.CORS_ALLOW_CREDENTIALS = True
-        settings.CORS_ORIGIN_ALLOW_ALL = False
-        settings.CORS_ALLOW_METHODS = ['OPTIONS']
-        settings.CORS_URLS_REGEX = '^.*$'
-        response = HttpResponse()
-        request_headers = {'HTTP_ORIGIN': 'http://foo.google.com'}
-        request = Mock(path='/', META=request_headers, method='OPTIONS')
-        processed = self.middleware.process_response(request, response)
-        self.assertEqual(processed.get(ACCESS_CONTROL_ALLOW_ORIGIN, None), 'http://foo.google.com')
-
-    def test_process_response_will_not_add_origin_when_domain_not_found_in_origin_regex_whitelist(self, settings):
-        settings.CORS_MODEL = None
-        settings.CORS_ORIGIN_REGEX_WHITELIST = ('^http?://(\w+\.)?yahoo\.com$', )
-        settings.CORS_ALLOW_CREDENTIALS = True
-        settings.CORS_ORIGIN_ALLOW_ALL = False
-        settings.CORS_ALLOW_METHODS = ['OPTIONS']
-        settings.CORS_URLS_REGEX = '^.*$'
-        response = HttpResponse()
-        request_headers = {'HTTP_ORIGIN': 'http://foo.google.com'}
-        request = Mock(path='/', META=request_headers, method='OPTIONS')
-        processed = self.middleware.process_response(request, response)
-        self.assertEqual(processed.get(ACCESS_CONTROL_ALLOW_ORIGIN, None), None)
-
-    def test_process_response_when_custom_model_enabled(self, settings):
+    def test_process_response_no_origin(self, get_settings):
+        with override_settings(CORS_MODEL=None, CORS_URLS_REGEX='^.*$'):
+            get_settings.return_value = CorsHeadersSettings(settings)
+            response = HttpResponse()
+            request = Mock(path='/', META={})
+            processed = self.middleware.process_response(request, response)
+            self.assertNotIn(ACCESS_CONTROL_ALLOW_ORIGIN, processed)
+
+    def test_process_response_not_in_whitelist(self, get_settings):
+        with override_settings(
+                CORS_MODEL=None, CORS_ORIGIN_ALLOW_ALL=False,
+                CORS_ORIGIN_WHITELIST=['example.com'], CORS_URLS_REGEX='^.*$'):
+            get_settings.return_value = CorsHeadersSettings(settings)
+            response = HttpResponse()
+            request = Mock(path='/', META={'HTTP_ORIGIN': 'http://foobar.it'})
+            processed = self.middleware.process_response(request, response)
+            self.assertNotIn(ACCESS_CONTROL_ALLOW_ORIGIN, processed)
+
+    def test_process_response_in_whitelist(self, get_settings):
+        with override_settings(
+                CORS_MODEL = None, CORS_ORIGIN_ALLOW_ALL = False,
```

Comment:
whitespaces around `=`, you have that a lot so I'll only comment once :sweat_smile: